source/screens/caseScreens/CaseOverview.tsx

```diff
@@ -139,13 +151,25 @@ const computeCaseCardComponent = (caseData, navigation, authContext, extra) => {
     (person) => person.personalNumber === authContext.user.personalNumber
   );
 
+  const completions = caseData?.details?.completions?.requested || [];
+
   const statusType = caseData?.status?.type || "";
-  const isNotStarted = statusType.includes("notStarted");
-  const isOngoing = statusType.includes("ongoing");
-  const isCompletionRequired = statusType.includes("completionRequired");
-  const isSigned = statusType.includes("signed");
-  const isClosed = statusType.includes("closed");
-  const isWaitingForSign = statusType.includes("active:signature:pending");
+  const isNotStarted = statusType.includes(NOT_STARTED);
+  const isOngoing = statusType.includes(ONGOING);
+  const isRandomCheckRequired = statusType.includes(
+    ACTIVE_COMPLETION_RANDOM_CHECK_REQUIRED_VIVA
```

Comment:
New constant name - remove _COMPLETION_

Reply:
I have created another task that changes this:
https://app.clickup.com/t/1ubxx06